store/project.go

```diff
@@ -326,10 +328,11 @@ func createProjectImpl(ctx context.Context, tx *Tx, create *api.ProjectCreate) (
 			tenant_mode,
 			db_name_template,
 			role_provider,
-			schema_change_type
+			schema_change_type,
```

Comment:
Need a reverse of https://github.com/bytebase/bytebase/pull/2677 unless we're sure to release in the next release (next week).